tracer/src/Datadog.Trace/Propagators/PropagationContext.cs

```diff
@@ -0,0 +1,23 @@
+﻿// <copyright file="PropagationContext.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#nullable enable
+
+namespace Datadog.Trace.Propagators;
+
+internal readonly struct PropagationContext
```

Comment:
Neat 👍🏼

Reply:
Thanks, but it was based on OpenTelemetry's implementation:
https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry.Api/Context/Propagation/PropagationContext.cs